build_scripts/docker/clean.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# This is a utility script to clean up after a failed or successful Docker
+# image build in order to speed up development and debugging
+
+DOCKER_DIR="$(realpath $(dirname $BASH_SOURCE[0]))"
+
+
+rm -rf "$HOME/git/monkey"
+rm -rf "$DOCKER_DIR/monkey"
+rm -rf "$DOCKER_DIR/tgz"
+rm "$DOCKER_DIR"/dk.monkeyisland.*.tar
+rm "$DOCKER_DIR"/infection_monkey_docker*.tgz
```

Comment:
Can we not delete the final artifact and instead force an override if another build is ran?

Reply:
The build artifacts will be overwritten if the build scripts are run again.